packages/devtools_extensions/lib/src/model.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'api.dart';
+
+/// Data model for a devtools extension event that will be sent and received
+/// over 'postMessage' between DevTools and the embedded extension iFrame.
+class DevToolsExtensionEvent {
```

Comment:
Can you explain in doc comment why the events are needed, and how they are supposed to be used?

Reply:
Added a reference to [DevToolsExtensionEventType]. This code is incomplete right now since we are starting with the `ping`, `pong` support, but in the future we will also support events like `connectToVmService`, maybe `showNotification` etc. When we support more events, the `DevToolsExtensionEventType` enum will be self documenting.